### ISSUE TITLE
Конструктор запроса, если реквизит составной, отображает не все типы, а только один

### ISSUE THREAD

**Issue by lrRatibor** [1C-Company/1c-edt-issues#1060]
### Описание ошибки

Если реквизит объект составной и в качестве типов указаны Ссылки (на справочник или документ), то в конструкторе запроса видно только один из типов. Получается можно выбрать Ссылка1.Наименование, а вот добраться до Ссылка2.Наименование уже не получится (кроме как руками писать запрос)

### Как воспроизвести

1. Создать у любого объекта реквизит составного типа (в качестве типов указать какие-нибудь справочники)
2. Создать запрос с этим объектом
3. Убедиться что у реквизита видно только один тип (один из справочников)

### Скриншоты

![Безымянный](https://github.com/1C-Company/1c-edt-issues/assets/125652316/227a2d9e-da0f-42fc-a33b-130563e5dbe5)
Сверху EDT, если раскрыть реквизит "объект", то сразу отображается реквизиты одой из ссылок
Снизу конфигуратор, тот же запрос, при раскрытии реквизита "объект" доступны все ссылки.

### Ожидаемое поведение

Если реквизит состоит из нескольких не ссылочных типов (строка, число, дата) отображать каждый смысла нет, в таком случае такое поведение логично (конфигуратор тоже не показывать отдельные типы).
Но если появляется хотя бы один ссылочный, то каждый ссылочный тип должен отображаться отдельно, что бы можно было добраться до реквизитов этого типа.

### Лог рабочей области

 -

### Версия 1С:EDT

Ruby 2023.1.0

### Операционная система

Windows

**Comment by adminimusRU** [comment]
На самом деле внутри составного типа ЕДТ показывает все реквизиты, входящие во все включенные типы. Реквизиты с одинаковыми названиями "схлопываются" в один. И в целом нет разницы, выбирать в поля Ссылка1.Наименование или `Ссылка2.Наименование `- в тексте запроса будет одно и то же `Объект.Наименование `.
Разница с конфигуратором чисто косметическая, он визуально группирует реквизиты составного типа по каждой таблице. В целом было бы хорошо иметь переключатель для возможности такого же отображения с группировкой по таблицам.

**Comment by lrRatibor** [comment]
Да, действительно был не прав,  в EDT присутствуют все реквизиты из разных типов, вот только это не всегда удобно когда они в одном месте. Например на скриншоте в зависимости от типа реквизита выбираются разные данные, в конфигураторе (сверху) сразу видно какие типы могут быть, а в EDT не понятно из каких типов состоит.
![image](https://github.com/1C-Company/1c-edt-issues/assets/125652316/6022fa7e-fb24-4ec3-934a-8632c6d4307a)

Тогда это скорее относиться к разделу пожелания.
